tls/s2n_config.h

```diff
@@ -125,6 +126,9 @@ struct s2n_config {
     uint8_t (*verify_host)(const char *host_name, size_t host_name_len, void *data);
     void *data_for_verify_host;
 
+    s2n_crl_lookup_fn crl_lookup;
+    void *data_for_crl_lookup;
+
```

Comment:
We generally call this the "context", not "data_for". I know verify_host calls it "data_for", but I'd stick to the more common naming.

Reply:
Fixed! I also changed `s2n_crl_lookup_context` to just `s2n_crl_lookup` to not conflict with the user context.